packages/components/package.json

```diff
@@ -118,6 +119,7 @@
     "react-swipeable": "^5.5.1",
     "react-tiny-popover": "^4.0.0",
     "react-transition-group": "^4.2.2",
+    "react-window": "^1.8.5",
```

Comment:
can react-window be removed from trader now?

Reply:
No. It's still used in `PositionsDrawer`.